supabase/tests/63_test_billing_period_stats_email.sql

```
@@ -0,0 +1,190 @@
+-- Ensures billing_period_stats is registered in cron_tasks and queues on anniversary day.
+BEGIN;
+
+SELECT plan(6);
+
+SELECT ok(
+    to_regprocedure('public.process_billing_period_stats_email()') IS NOT NULL,
+    'process_billing_period_stats_email exists'
+);
+
+SELECT ok(
+    to_regprocedure('public.get_org_credits_used_in_period(uuid, timestamptz, timestamptz)') IS NOT NULL,
```

SQLFluff (4.2.2)

[error] Line is too long (105 > 80).

(LT05)